apps/gallery/js/ImageEditor.js

```diff
@@ -226,7 +226,7 @@ var exposureSlider = (function() {
     if (!dragStart)
       dragStart = currentExposure;
 
-    setExposure(dragStart + exposureDelta);
+    setTimeout(function() { setExposure(dragStart + exposureDelta); });
```

Comment:
Couldn't place why  we need setTimeout here? Is  it here to ensure function execution queueing and  helping with slider performance.